apps/ccd/automation/kustomization.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../ccd-ac-int-data-store-api/image-repo.yaml
-  - ../ccd-ac-int-data-store-api/demo-image-policy.yaml
+  # - ../ccd-ac-int-data-store-api/image-repo.yaml
+  # - ../ccd-ac-int-data-store-api/demo-image-policy.yaml
```

Comment:
Hey @NickAzureDevops i see commented out code `#` is this as expected? comments says uncommented out code

Reply:
meant to be commented out and not the other way around my bad

Reply:
Oh i see no longer needed so removing